stripe/src/main/java/com/stripe/android/model/PaymentMethod.kt

```diff
@@ -117,15 +117,15 @@ data class PaymentMethod internal constructor(
     @Parcelize
     enum class Type constructor(
         @JvmField val code: String,
-        @JvmField val isReusable: Boolean = true
+        @JvmField val isReusable: Boolean
     ) : Parcelable {
-        Card("card"),
-        CardPresent("card_present"),
+        Card("card", isReusable = true),
+        CardPresent("card_present", isReusable = false),
         Fpx("fpx", isReusable = false),
-        Ideal("ideal"),
-        SepaDebit("sepa_debit"),
-        AuBecsDebit("au_becs_debit"),
-        BacsDebit("bacs_debit"),
+        Ideal("ideal", isReusable = false),
```

Comment:
This was a bug before?

Reply:
Technically yes, but in practice this was only really used in our basic integration screens, and we don't support Ideal there.